PaTTAN Room Displays/PaTTAN Room Displays/App.xaml.cs

```diff
@@ -1,11 +1,28 @@
-﻿namespace PaTTAN_Room_Displays;
+﻿using System.Threading.Tasks;
+
+namespace PaTTAN_Room_Displays;
 
 public partial class App : Application
 {
+	RoomDataService roomDataService = new RoomDataService();
+	public static List<Meeting> meetingList = new List<Meeting>();
 	public App()
 	{
 		InitializeComponent();
+		getData();
 
-		MainPage = new MainPage();
+		if(meetingList.Count > 0)
+        {
+			MainPage = new MainPage();
+        } else
+        {
+			MainPage = new NoMeetingsPage();
+        }
+		
 	}
+
+	private void getData()
```

Comment:
Why is this single line a separate function?

Reply:
that gets removed in a later PR.